packages/flutter_tools/lib/executable.dart

```diff
@@ -59,6 +59,7 @@ import 'src/usage.dart';
 ///
 /// This function is intended to be used from the `flutter` command line tool.
 Future<Null> main(List<String> args) async {
+  SigintProcessSignal.instance.init();
```

Comment:
Why initialize this here versus just lazily initializing it when we first call `watch()`?